contrib/repro_test.bzl

```diff
@@ -129,12 +129,24 @@ def _impl(ctx):
     extract_path = "/img_outs"
     img_tar = img_name + ".tar"
     img_digest = img_name + ".digest"
+    host_outs_path1 = "/tmp/" + name + "/bazel_out1"
```

Comment:
As mentioned offline, lets do our best to not have to mount anything under /tmp directory

Reply:
#1003 solves this.